docs/concepts/security.md

```diff
@@ -0,0 +1,60 @@
+# Security
+
+This document explains the security aspects of Kong Ingress Controller.
+
+Kong Ingress Controller communicates with Kubernetes API-server and Kong's
+Admin API. APIs on both sides offer authentication/authorization features
+and the controller integrates with them gracefully.
+
+## Kubernetes RBAC
+
+Kong Ingress Controller is deployed with RBAC permissions as explained in
+[deployment](deployment.md) document.
+It has read and list permissions on most resources but requires update
+and create permission for a few resources to provide seamless integration.
+The permissions can be locked down further if needed depending on the specific
+use-case.
+This RBAC policy is associated with a ServiceAccount in Kubernetes, which
+takes of the Controller authenticating and authorizing against Kuberentes
+API-server.
+
+## Kong Admin API protection
+
+Kong's Admin API is used to control configuration of Kong and proxying behavior.
+If an attacker gets access to Kong's Admin API, Kong will be compromised
+and all bets are off at that point. Hence, it is important that the deployment
+ensures that the likelyhood of this happening is minimized.
+
+In the example deployements, the Controller and Kong's Admin API communicate
+over the loopback(lo) interface of the pod. There is no authorization or
+authentication being done over the loopback listner.
```

Comment:
```suggestion
authentication taking place over the loopback listener.
```

Reply:
if possible to say _what_ is not doing authentication or authorization, it'd be better to make the sentence active. 

"_X_ is not performing authentication or authorization over the loopback listener."